plugins/plugins/frontend/public/localization/plugins.properties

```diff
@@ -82,25 +82,25 @@ configs.logs.warn = Warning Level
 configs.logs.error = Error Level
 configs.logs.default = Default Level
 
-configs.help.frontend-production = If enabled, serves minified, combined files
+configs.help.frontend-production = If enabled, serves minified, combined files. (what is the cause of disabling or enabling this? Any potential harms?)
```

Comment:
I am sure, this is a comment, rather than a change :)

Reply:
Right, that was my comment for you in fact.. There is a need for a better explanation if you think it helps?

Reply:
:) Sorry for that. I'll be more carefull. I just copied it :(

Reply:
Initial load of dashboard should be faster, due to smaller files and smaller file amount, but when developing a plugin, you need to regenerate them to see changes

Reply:
No probs, thats what reviews are for :)